eventmesh-registry-plugin/eventmesh-registry-zookeeper/src/main/java/org/apache/eventmesh/registry/zookeeper/constant/ZookeeperConstant.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.registry.zookeeper.constant;
+
+import java.nio.charset.Charset;
+
+/**
+ * ZookeeperConstant.
+ */
+public class ZookeeperConstant {
+
+    public static final String NAMESPACE = "eventmesh";
+
+    public static final String IP_PORT_SEPARATOR = ":";
+
+    public static final String PATH_SEPARATOR = "/";
+
+    public static final Charset CHARSET_UTF8 = Charset.forName("utf-8");
```

Comment:
You can directly use `StandardCharsets.UTF_8` rather than define it again in this class.